Error: spawn phantomjs ENOENT

Trying to find the root cause of https://github.com/juliangruber/browser-run/issues/102

```
~/forks/phantomjs-stream master 6s
❯ npm t

> phantomjs-stream@1.1.1 test /Users/zeke/forks/phantomjs-stream
> make test

TAP version 13
# execute
events.js:154
      throw er; // Unhandled 'error' event
      ^

Error: spawn phantomjs ENOENT
    at exports._errnoException (util.js:890:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:182:32)
    at onErrorNT (internal/child_process.js:348:16)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9)
make: *** [test] Error 1
npm ERR! Test failed.  See above for more details.

~/forks/phantomjs-stream master
❯ node -v
v5.10.1
```

--- a/package.json
+++ b/package.json
@@ -8,6 +8,7 @@
   "repository": "juliangruber/phantomjs-stream",
   "devDependencies": {
     "concat-stream": "^1.5.0",
+    "phantomjs-prebuilt": "^2.1.7",
     "tape": "4.5.1"
   },
   "license": "MIT",

